llama_stack/providers/remote/safety/bedrock/bedrock.py

```diff
@@ -93,7 +89,7 @@ async def run_shield(
         )
 
         response = self.bedrock_runtime_client.apply_guardrail(
-            guardrailIdentifier=shield_params["guardrailIdentifier"],
+            guardrailIdentifier=shield.identifier,
```

Comment:
this should be the `provider` specific id right? so ideally, `shield.provider_resource_identifier`?

Reply:
ah you are right. changing.